docs/guide/writing-tasks.md

```diff
@@ -408,19 +408,19 @@ build_and_test_task:
     It is possible to refer to the artifacts of the current build directly:
 
     ```
-    https://api.cirrus-ci.com/v1/artifact/build/<CIRRUS_BUILD_ID>/<ARTIFACTS_NAME>.zip
+    https://api.cirrus-ci.com/v1/artifact/build/<CIRRUS_BUILD_ID>/<ARTIFACTS NAME>.zip
     ```
 
     Note that if several tasks are uploading artifacts with the same name then the ZIP archive from the above link will
     contain merged content of all artifacts. It's also possible to refer to an artifact of a particular task within a build
     by name:
 
     ```
-    https://api.cirrus-ci.com/v1/artifact/build/<CIRRUS_BUILD_ID>/<TASK_NAME>/<ARTIFACTS_NAME>.zip
+    https://api.cirrus-ci.com/v1/artifact/build/<CIRRUS_BUILD_ID>/<TASK NAME OR ALIAS>/<ARTIFACTS NAME>.zip
     ```
 
-    It's also possible to download a particular file of an artifact and not the whole archive by using `<ARTIFACTS_NAME>/<PATH>`
-    instead of `<ARTIFACTS_NAME>.zip`.
+    It's also possible to download a particular file of an artifact and not the whole archive by using `<ARTIFACTS NAME>/<PATH>`
```

Comment:
If you're changing `ARTIFACTS_NAME` to `ARTIFACTS NAME` (why, btw), maybe change it everywhere then?

Reply:
Agreed, let's stick with `ARTIFACTS_NAME`. Fixed in 40c87af